ydb/core/fq/libs/row_dispatcher/format_handler/format_handler.h

```diff
@@ -15,7 +15,7 @@ class IClientDataConsumer : public TThrRefBase {
     using TPtr = TIntrusivePtr<IClientDataConsumer>;
 
 public:
-    virtual TVector<TSchemaColumn> GetColumns() const = 0;
+    virtual const TVector<TSchemaColumn> GetColumns() const = 0;
```

Comment:
Sorry, я имел в виду (возврат константной не-ссылки это, кмк, no-op)
```suggestion
    virtual const TVector<TSchemaColumn>& GetColumns() const = 0;
```